pkg/controller/proxyconfig/controller.go

```diff
@@ -82,6 +83,10 @@ type ReconcileProxyConfig struct {
 func (r *ReconcileProxyConfig) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	es, err := k8shandler.GetElasticsearchCR(r.client, request.Namespace)
 	if err != nil {
+		if errors.Is(err, k8shandler.ErrEsListEmpty) {
```

Comment:
i believe there is an `errors.IsNotFound` that we can use to short-hand this... no?